docs/sources/panels-visualizations/configure-thresholds/index.md

```diff
@@ -25,6 +25,8 @@ In dashboards, a threshold is a value or limit you set for a metric that's refle
 Using thresholds, you can color grid lines and regions in a time series visualization:
 ![Time series visualization with green, blue, and purple threshold lines and regions](/media/docs/grafana/panels-visualizations/screenshot-thresholds-lines-regions-v10.4.png)
 
+{{< docs/play title="Threshold example" url="https://play.grafana.org/d/000000167/" >}}
```

Comment:
I would move this down to after line 41 to avoid putting it between images that are part of the same thought and not having four blocks in a row on the page.